python/arcticdb/version_store/_store.py

```diff
@@ -332,6 +332,9 @@ def _try_normalize(
         dynamic_schema = self.resolve_defaults(
             "dynamic_schema", self._lib_cfg.lib_desc.version.write_options, False, **kwargs
         )
+        empty_types = self.resolve_defaults(
```

Comment:
This isn't quite right, as the C++ layer only checks the lib config, not the kwargs. I don't think this needs to be a kwarg, so sufficient to pass an empty dict here

Reply:
Agreed, we don't want anyone to be able to mix and match within a library, that would be bad news